src/app/src/components/FacilityListsTable.jsx

```diff
@@ -11,6 +11,21 @@ import TableCell from '@material-ui/core/TableCell';
 import { facilityListPropType } from '../util/propTypes';
 import { makeFacilityListItemsDetailLink } from '../util/util';
 
+import COLOURS from '../util/COLOURS';
+
+const facilityListsTableStyles = Object.freeze({
+    inactiveListStyles: Object.freeze({
+        cursor: 'pointer',
+    }),
+    activeListStyles: Object.freeze({
+        backgroundColor: 'rgb(222, 242, 254)',
+        outlineWidth: '0.75px',
+        outlineStyle: 'solid',
+        outlineColor: COLOURS.NAVY_BLUE,
```

Comment:
In the mockup this was `#1a9fe3`:

![image](https://user-images.githubusercontent.com/1928955/54045347-5c8f2000-419f-11e9-9c78-af3e37a1bdad.png)

Reply:
:+1: I'll change this one before merging, also.